README.md

```diff
@@ -1,24 +1,67 @@
-# README
+# Budget App
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+> A Mobile Web App made with Rails that takes care of your expenses. You can have a record of your expenses by date, category, amount, and others. Has feature and system tests with RSpec and Capybara. Implements authentication with Devise. Deployed to Heroku. Uses PostgreSQL and ActiveRecord.
 
-Things you may want to cover:
+# Live Demo
+For a live demo of the project [click here](https://obscure-brushlands-44462.herokuapp.com). (view on mobile for a better experience)
```

Comment:
- [ ] The deployed app doesn't work after sign up. Since you are the application owner, check the logs to address the problem, remember that this is one of the project requirements.

- [ ] The video recording was good, but you didn't address all the requirements of the recording, kindly record another video and highlight a piece of code you are proud of.